_setup_support.py

```diff
@@ -412,7 +412,7 @@ def package_path(path, filters=()):
 
 
 Please build BokehJS by running setup.py with the `--build-js` option.
-  Dev Guide: https://bokeh.pydata.org/docs/dev_guide.html#bokehjs.
+  Dev Guide: https://bokeh.pydata.org/en/latest/docs/dev_guide/setup.html.
```

Comment:
This should be changed to *docs.bokeh.org* and in fact that is true everywhere, if you wanted to look for and update old docs URLs as well.

Reply:
Ah yes, I was wondering about the two different domains.

If *docs.bokeh.org* is the preferred domain, wouldn't it make sense to redirect all traffic from *bokeh.pydata.org* to that site? I can imagine it would have SEO advantages and might be clearer for users. Or is there some technical reason preventing this?

Reply:
@slokhorst We already do to that, so that all the (many) old links to the *bokeh.pydata.org* domain still function. But we should update things to use *docs.bokeh.org* wherever we can going forward.

Reply:
I mean like a HTTP 301 redirect. That way if one navigates to the old domain they will be redirected to the new one. That will also cause search engines to update and adopt the new domain. (now, if I search for "Bokeh docs" on Google, it only shows me *bokeh.pydata.org* links because it thinks it is still an active site)

Reply:
Well, I just tried to add a 301 page rule on Cloudflare (exactly following some other ones that are already in place for other old pydata subdomains) and I seem to have taken the docs offline...

Reply:
OK can you tell me what you are seeing now? My local system seems to have gotten a bad redirect that I can't un-cache

Reply:
OK I *think* things are working now? I did not realize Cloudflare page rules need some proxied ("orange cloud") DNS entry to function. Getting what appear to be correct results at http://www.redirect-checker.org/index.php tho my local system still seems to be unhappy.

Reply:
Awesome! Things are working perfectly now from my side. Both http and https traffic is being redirected nicely:

```
$ curl -I http://bokeh.pydata.org/
HTTP/1.1 301 Moved Permanently
...
location: https://docs.bokeh.org/
```

```
$ curl -I https://bokeh.pydata.org/
HTTP/2 301 
...
location: https://docs.bokeh.org/
```

If you're still having issues, let me know if I can help debug!